var/spack/repos/builtin/packages/mvapich/package.py

```diff
@@ -285,11 +288,6 @@ def configure_args(self):
         else:
             args.append("--disable-registration-cache")
 
-        ld = ""
```

Comment:
Why are all of these lines being removed? @nkgh77

Reply:
Oh, sorry. This is an accident. How can I recover from the mistake?

Reply:
If your branch is based off of an older branch try merging the newest version into your branch and then fix any conflicts that arise.

Reply:
Eventually, I did it manually, but I believe the changes shown in DIFF are all my intentions.